CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - `sapmreceiver`: Fix issue where component instance use in multiple pipelines leads to start failures (#11518)
 - `signalfxreceiver`: Fix issue where component instance use in multiple pipelines leads to start failures (#11513)
 - `splunkhecreceiver`: Fix issue where component instance use in multiple pipelines leads to start failures (#11517)
+- `skywalkingreceiver`: Fix skywalking traceid and spanid convertion (#11562)
```

Comment:
Please use the new way of adding changelog entries
👉 https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-a-changelog-entry